docker/spec/dependabot/docker/update_checker_spec.rb

```diff
@@ -960,4 +990,17 @@
       end
     end
   end
+
+  private
+
+  def stub_same_sha_for(*tags)
```

Comment:
Is it convention to put private test helpers at the bottom of the spec files? In other language ecosystems, I'm more used to them being at the top.

Reply:
No, not really, I think my personal style here is pretty rare actually. Normally people would move these into a `let` at the top indeed. Should I change it?

Reply:
Actually, since `let` does not take parameters, I'm not really sure where people put this kind of test helper 😅. I'll leave this as is.